proposals/0777-remove-legacy-element-inspector-features.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Removing the "Perf" and "Network" tabs from the in-app Element Inspector
+author:
+- Alex Hunt <huntie@meta.com>
+date: 2024-04-04
+---
+
+# RFC0777: Removing the "Perf" and "Network" tabs from the in-app Element Inspector
+
+## Summary
+
+This RFC identifies two sub-features of the in-app Element Inspector which we believe are **not worth reimplementing** under the New Architecture.
```

Comment:
Q: Can you elaborate also on the other 2 tabs - Inspect and Touchables? Why are we keeping them? Are people actually using them?